Library/Formula/vdirsyncer.rb

```diff
@@ -0,0 +1,91 @@
+class Vdirsyncer < Formula
+  desc "Synchronize calendars and contacts"
+  homepage "https://github.com/untitaker/vdirsyncer"
+  url "https://pypi.python.org/packages/source/v/vdirsyncer/vdirsyncer-0.6.0.tar.gz"
+  sha256 "0d2a9677b086cfbe5fe5e7cb5e55db4c1afab62cb5dc56aeaff6e95d34bf60d5"
+  head "https://github.com/untitaker/vdirsyncer.git"
+
+  option "without-keyring", "Build without python-keyring support"
+
+  depends_on :python
```

Comment:
This should be `depends_on :python if MacOS.version <= :snow_leopard` AFAIK. Otherwise we end up making everyone who installs this from a bottle install our Python first.